codec-http/src/test/resources/jquery-3.5.1.min.js

```diff
@@ -0,0 +1,2 @@
+/*! jQuery v3.5.1 | (c) JS Foundation and other contributors | jquery.org/license */
```

Comment:
@slandelle we will need to update our NOTICE file and add the license of jquery to the license folder.

Reply:
This is just a sample. I'm going so switch to a different file, eg Netty's HTML home page so we don't have such dependency and don't end up raising security bots alarms when there's a security issue with a jQuery version.